### PR TITLE
Create a Readme for Pikachus

### PR DESCRIPTION
**Pull request by JamesKerryBarnard**
Pikachu (the best character) will not be able to read the main README, so i propose creating a Pikchu specific Readme.

### CODE REVIEW

**Comment by chuckydev** [comment]
No.

**Comment by JamesKerryBarnard** [comment]
Please Review this

**Comment by DeanIsseyegh** [comment]
Pikachus ftilt comes out on frame 6, and his dthrow comboes into nair loops up to around 40% where it can be finished off with a dsmash.

You haven't included ANY of this info in your PR.

File: PIKAREADME.MD [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PIKA PIKA PIKA PIKA PIKA PIKAAAA
+
+
+CHUUUUUUUUUU
```
Comment (chuckydev) [inline PIKAREADME.MD:4]: Needs a picture of Pikachu

**Review by chuckydev — CHANGES_REQUESTED** [review]

File: PIKAREADME.MD (hunk 1 above)
Comment (JamesKerryBarnard) [reply PIKAREADME.MD:4]: Agreed. +1 a pika pic

**Comment by JamesKerryBarnard** [comment]
Hello @DeanIsseyegh I have added a pikachu pic

**Pull request closed without merging**